developer/src/kmc-kmn/src/compiler/compiler.ts

```diff
@@ -322,12 +273,17 @@ export class KmnCompiler implements KeymanCompiler, UnicodeSetParser {
 
     outfile = outfile ?? infile.replace(/\.kmn$/i, '.kmx');
 
-    (globalThis as any)[this.callbackID] = {
-      message: this.compilerMessageCallback,
-      loadFile: this.loadFileCallback
-    };
+    const compiler = this;
+    const wasm_callbacks = Module.WasmCallbackInterface.implement({
+      message: function(message: KmnCompilerResultMessage) {
+        compiler.callbacks.reportMessage(mapErrorFromKmcmplib(message.lineNumber, message.errorCode, message.message));
+      },
+      loadFile: function(filename: string, baseFilename: string): number[] {
```

Comment:
Was `loadFileCallback`.  Obviously, it's been dramatically simplified from the original.

(I didn't realize this before; only learned via paired review.  Documenting this for future reference.)